diff -c -r a/frontend_qt/mainwindow.cpp b/frontend_qt/mainwindow.cpp
*** a/frontend_qt/mainwindow.cpp	2022-05-24 22:08:51.000000000 +0200
--- b/frontend_qt/mainwindow.cpp	2022-06-25 21:18:23.974615664 +0200
***************
*** 27,32 ****
--- 27,33 ----
  #include <QClipboard>
  #include <QMimeData>
  #include <QColor>
+ #include <QtGlobal>
  #include <QListView>
  #include <QShortcut>
  #include <QMenu>
***************
*** 2636,2644 ****
      m_copyTIFShortcut->setEnabled(enabled);
  }
  
! void MainWindow::copy_to_clipboard(const QString &filename, const QString& name, const char *mimeType)
  {
      QClipboard *clipboard = QGuiApplication::clipboard();
  
      if (!m_bc.bc.save_to_file(filename)) {
          return;
--- 2637,2647 ----
      m_copyTIFShortcut->setEnabled(enabled);
  }
  
! void MainWindow::copy_to_clipboard(const QString &in_filename, const QString& name, const char *mimeType)
  {
      QClipboard *clipboard = QGuiApplication::clipboard();
+     
+     auto filename = QString::fromLocal8Bit(qgetenv("XDG_RUNTIME_DIR")).append("/app/uk.org.zint.zint-qt/").append(in_filename);
  
      if (!m_bc.bc.save_to_file(filename)) {
          return;

